jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebAppContext.java

```diff
@@ -1271,6 +1274,16 @@ protected void stopWebapp() throws Exception
         super.stopContext();
     }
 
+    /**
+     * Prevent the temp directory from being deleted during the normal stop sequence, and require that
+     * super.deleteTempDirectory() is explicitly called at the right time, after the webapp classloader is closed
+     */
+    @Override
+    protected void deleteTempDirectory() throws Exception
+    {
+        //intentionally left blank
+    }
```

Comment:
It like this is a little bit of a hack.  It would be better if another method was nulled out and that `deleteTempDirectory` always did just that.

So we need another level of delegation... and a good name to call it... stopContext has already been taken.  let me think a bit....